Sources/MapboxCoreNavigation/MapboxRoutingProvider.swift

```diff
@@ -209,7 +209,7 @@ public class MapboxRoutingProvider: RoutingProvider {
         let directionsUri = settings.directions.url(forCalculating: options).removingSKU().absoluteString
         var requestId: RequestId!
         
-        requestId = router.getRouteForDirectionsUri(directionsUri) { [weak self] (result, origin) in
+        requestId = router.getRouteForDirectionsUri(directionsUri, options: GetRouteOptions(timeoutSeconds: nil)) { [weak self] (result, origin) in
```

Comment:
Should we set a default timeout for the route request?  @mapbox/navigation-ios

Reply:
I think we have 60 seconds timeout in Directions, but since this setting allows a 'default' nil - we should probably use that.